README.md

```diff
@@ -81,48 +83,58 @@ MARCO FEATURES
 
  - Has a simple theme system and a couple of extra themes come with it.
    Change themes via gsettings:
+     ```
```

Comment:
Those aren't MATE themes and they don't support gtk+-3.
Please use themes from https://github.com/mate-desktop/mate-themes
Our default are the Menta themes.
A dark one called BlackMATE and former clearlooks are called Traditional themes.
Thanks